index.html

```diff
@@ -12050,9 +12057,14 @@ <h2>Context Definitions</h2>
     <code>@container</code>,
     <code class="changed">@context</code>,
     <code class="changed">@prefix</code>,
+    <code class="changed">@propagate</code>, or
```

Comment:
I did remove `@propagate` from this list earlier (b4dae4d), because I don't see anything the API where it is expected in an expanded term definition...

Am I missing something?

Reply:
No, I you’re right, only a context definition can contain `@propagate`.

Reply:
No, I you’re right, only a context definition can contain `@propagate`.